modules/interop-specs2/test/src/weaver/interop/specs2/MatchersSpec.scala

```diff
@@ -0,0 +1,25 @@
+package weaver.interop.specs2
```

Comment:
I'm waging a war against empty packages, I think it's an arbitrary javaism that makes apis genuinely harder to use, by making the user having to refer themselves to the documentation more often. That's (partly) why the top package is `weaver` as opposed to `com.disneystreaming.weaver`

I think the following is fine : 

```scala
import weaver.specs2 
```

If you're concerned about package collision (which can be a genuine concern depending on the UX of the library you're trying to interop with), I think `weaver.specs2compat` would do (the zio package is `ziocompat` for this reason). It's auto-completion friendly.